python/pysrc/kaskada/windows.py

```diff
@@ -55,9 +60,60 @@ def yearly() -> Since:
         return Since(predicate=lambda domain: Timestream._call("yearly", domain))
 
 
+@dataclass(frozen=True)
+class Tumbling(Window):
+    """A series of non-overlapping windows that reset each time a predicate evaluates to true.
```

Comment:
Change the terms here to match more of the tumbling window concepts. I don't think that is normally described as "reset each time a predicate evaluates to true"